support/src/main/java/ru/nikartm/support/model/Badge.java

```diff
@@ -2,7 +2,7 @@
 
 import android.graphics.Typeface;
 import android.graphics.drawable.Drawable;
-import android.support.annotation.Nullable;
+import androidx.annotation.Nullable;
```

Comment:
Have you attempt to use this https://gist.github.com/mochadwi/d233e4996f3e498f08f57d073e366c9b to migrate all possible import including the build.gradle & XML

The gist is inspired by danlew (trello android engineer) 

@anangkur

Reply:
I just use the `migrate to androidx` menu built-in android-studio mas @mochadwi should I try running your script?

Reply:
Agreed

Reply:
alright will update this thread when I finished run the script thank you mas @mochadwi 🙏🏽 .

Reply:
done mas @mochadwi